dist/src/main/resources/application.properties

```diff
@@ -103,3 +103,6 @@ management.metrics.distribution.percentiles-histogram.http.server.requests=true
 management.metrics.distribution.percentiles.http.server.requests=0.5,0.95,0.99
 
 camunda.security.multiTenancy.checksEnabled=${zeebe.broker.gateway.multiTenancy.enabled:false}
+
+# Unified Configuration
+camunda.data.secondary-storage.elasticsearch.url=http://localhost:9200
```

Comment:
:x: I think this should be removed and should be only added in elasticsearch profile.